Bar.js

```diff
@@ -109,18 +124,23 @@ export default class ProgressBar extends Component {
       ...restProps
     } = this.props;
 
-    const innerWidth = width - (borderWidth * 2);
+    const viewWidth = this.state.measuredLayout.width;
+    const viewHeight = this.state.measuredLayout.height;
+    const innerWidth = viewWidth - (borderWidth * 2);
+    const innerHeight = viewHeight - (borderWidth * 2);
+    console.log(viewWidth+ " "+viewHeight)
     const containerStyle = {
-      width,
+      height: React.Children.count(children) > 0 ? null : viewHeight, // Uses specified height or wraps the children content
```

Comment:
This check makes sure the height does not default to wrapping the content since most of the time children will be empty